spec/lib/ruy/time_pattern_spec.rb

```diff
@@ -427,4 +427,13 @@
     it_behaves_like 'time pattern comparison'
   end
 
+  describe 'method delegation' do
+    it 'delegates to its UTC based Time instance' do
+      expect(subject.gmt_offset).to eq(subject.utc.gmt_offset)
+    end
+
+    it 'delegates to super by default' do
+      expect(subject.class).to eq(Ruy::TimePattern)
+    end
```

Comment:
I'm confused by the test and/or the description. How does this test that methods are delegated to super?